tutorials/deploy/deploy-petclinic-container-14.1.2/README.md

```diff
@@ -1,24 +1,24 @@
 # Deploy Spring Framework PetClinic to WLS 14.1.2 running in a container
 
-This tutorial demonstrates how to deploy the [Spring Framework PetClinic example](/samples/spring-framework-petclinic-12.2.1.4/) application on WebLogic Server 14.1.2 running in a container (Docker or Podman or other of preference).
+This tutorial demonstrates how to deploy the [Spring Framework PetClinic](/samples/spring-framework-petclinic-12.2.1.4/) example application on WebLogic Server 14.1.2 running in a container (Docker or Podman, or other preference).
 
 ## Prerequisites
 
 Before starting this tutorial, make sure you have the following:
 
 - [Docker](https://docs.docker.com/get-docker/) or [Podman](https://podman.io/getting-started/installation) installed.
-- Access to the [https://container-registry.oracle.com/ords/ocr/ba/middleware/weblogic](https://container-registry.oracle.com/ords/ocr/ba/middleware/weblogic) or [WebLogic Server 14.1.2](https://www.oracle.com/middleware/technologies/weblogic-server-installers-downloads.html) image available in your local repository.
-- The [Spring Framework PetClinic example](/samples/spring-framework-petclinic-12.2.1.4/).
+- Access to [Oracle WebLogic Server 14.1.2.0 Container Images](https://container-registry.oracle.com/ords/ocr/ba/middleware/weblogic) or have the [WebLogic Server 14.1.2](https://www.oracle.com/middleware/technologies/weblogic-server-installers-downloads.html) image available in your local repository.
+- The [Spring Framework PetClinic](/samples/spring-framework-petclinic-12.2.1.4/) example.
 
-## Spring Framework PetClinic Example
+## Spring Framework PetClinic example
 
-We'll step through this tutorial using the Spring Framework PetClinic example that is available on this GitHub repo: [`spring-framework-petclinic-12.2.1.4` sample](../../../samples/spring-framework-petclinic-12.2.1.4/). This is a fork of the approved [fork](https://github.com/spring-petclinic/spring-framework-petclinic/tree/5.3.x) from Spring Team, a Petclinic version with a plain Spring Framework 5.3.x configuration and with a 3-layer architecture (i.e. presentation --> service --> repository) and was modified to run on WebLogic 12.2.1.4 and 14.1.2.
+We'll step through this tutorial using the Spring Framework PetClinic example that is available on this GitHub repo: [`spring-framework-petclinic-12.2.1.4`](../../../samples/spring-framework-petclinic-12.2.1.4/). This is a fork of the approved [fork](https://github.com/spring-petclinic/spring-framework-petclinic/tree/5.3.x) from the Spring Team, a PetClinic version with a plain Spring Framework 5.3.x configuration and with a 3-layer architecture (i.e. presentation --> service --> repository) and was modified to run on WebLogic 12.2.1.4 and 14.1.2.
 
```

Comment:
I think we should include a note(TIP) saying that optionally you migrate the app to use with 15.1.1 Beta using the migration tutorial.

> [!TIP]
> If you want to deploy the app to the WebLogic 15.1.1 beta ...